shared/common-adapters/clickable-box.js.flow

```diff
@@ -5,8 +5,8 @@ import React, {Component} from 'react'
 export type Props = {
   style?: Object,
   onClick: () => void,
-  hoverColor?: string,
-  backgroundColor?: string,
+  hoverColor?: ?string,
```

Comment:
yeah, i want to dynamically set this, like {hoverColor: this.state.something ? 'value' : null} but it won't let me do that without allowing ?string